### ISSUE TITLE
relative abundance to counts

### ISSUE THREAD

**Issue by pameubillag** [evolgeniusteam/GMrepoProgrammableAccess#4]
Hello, and thanks for your repository. 
I have been looking at the data, and I see that all abundances accessible are relative abundance. I haven't found any way to download counts data. Is count data available?? 
Also, as I am interested in count abundances, looking for a way to calculate it from relative abundance, I found a "nr_reads_sequenced," and I wonder what that number represents. For example, when it is metagenomic data, nr_reads_sequenced means all the oligonucleotides sequenced or bins?

**Comment by evolgeniusteam** [comment]
No, counts data are not available. Yes, you can obtain counts data using the relative abundances and the nr_reads_sequenced of the corresponding samples/runs. I have seen publications doing so. However, I never tried this and thus am in no position to judge whether it is a good or not.

The “Nr_reads_sequenced” always means the total number of sequencing reads in a sample/run, for both 16S and mNGS data.

Let me know if you need further assistance.

**Comment by pameubillag** [comment]
Thanks for the clarification.
